lib/feature_management.rb

```diff
@@ -48,8 +48,9 @@ def self.enable_identity_verification?
     Figaro.env.enable_identity_verification == 'true'
   end
 
-  def self.reveal_usps_code?
-    Rails.env.development? || current_env_allowed_to_see_usps_code?
+  def self.reveal_usps_flow?
+    env = Rails.env
+    env.development? || env.test? || current_env_allowed_to_see_usps_code?
```

Comment:
Re: the first part of the comment, I thought our feature flags were entirely handled in the feature management class. I'll look at the `enable_identity_verification` flag for insight!